src/Tracy/BlueScreen.php

```diff
@@ -68,6 +68,24 @@ public function render($exception)
 
 
 	/**
+	 * @param \Exception|\Throwable
+	 * @param string file path
+	 * @return void
+	 */
+	public function renderToFile($exception, $file)
+	{
+		if ($handle = @fopen($file, 'x')) { // @ file may already exist
```

Comment:
File should be opened twice, via `@fopen($file, 'x')` in logger and `fopen($file, 'w')` in Bluescreen.